integral_view/views/common.py

```diff
@@ -18,6 +18,14 @@
 def view_system_info(request):
     return_dict = {}
     try:
+        if "ack" in request.GET:
+            if request.GET["ack"] == "time":
```

Comment:
change these ack to just one 'system_date_time_set' because you may not know if one or both have been set.